compiler/test/compilable/traits.d

```diff
@@ -316,3 +316,7 @@ extern(C++, `inst`)
 mixin GetNamespaceTestTemplatedMixin!() GNTT;
 
 static assert (__traits(getCppNamespaces, GNTT.foo) == Seq!(`inst`,/*`decl`,*/ `f`));
+
+int[1] arr;
+// test parsing a non-compile-time expression
```

Comment:
Misleading comment. The parsing bug has nothing to do with whether the expression is "compile-time" or not.
```suggestion
// test that index assignment parses as an expression, not a type
```